plugins/pulp_docker/plugins/distributors/publish_steps.py

```diff
@@ -348,7 +411,11 @@ def process_main(self, item=None):
         :param item: The tag to process
         :type  item: pulp_docker.plugins.models.Tag
         """
-        manifest = models.Manifest.objects.get(digest=item.manifest_digest)
+        #TODO check if rsync works
+        try:
+            manifest = models.Manifest.objects.get(digest=item.manifest_digest)
+        except:
```

Comment:
I assume you'll catch a specific exception class before this becomes a finished PR.